gradle/libs.versions.toml

```diff
@@ -0,0 +1,213 @@
+[versions]
+kotlin = "1.8.20"
+
+kotlinx-serialization = "1.5.1"
+ktoml = "0.5.0"
+
+kotlinx-coroutines = "1.7.1"
+
+asm = "9.5"
+
+konf = "1.1.2"
+
+fuel = "2.3.1"
+
+junit-jupiter = "5.9.2"
+
+grgit = "4.1.1"
+
+
+[plugins]
+kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
+kotlin-serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin" }
+
+axion-release = { id = "pl.allegro.tech.build.axion-release", version = "1.15.0" }
+
+shadow = { id = "com.github.johnrengelman.shadow", version = "8.1.1" }
+
+dokka = { id = "org.jetbrains.dokka", version.ref = "kotlin" }
+
+compose = { id = "org.jetbrains.compose", version = "1.4.0" }
+
+
+[libraries]
+
+# Kotlin stdlib
+kotlin-stdlib = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib", version.ref = "kotlin" }
+kotlin-reflect = { group = "org.jetbrains.kotlin", name = "kotlin-reflect", version.ref = "kotlin" }
+
+# kotlinx.serialization
+kotlinx-serialization-core = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-core", version.ref = "kotlinx-serialization" }
+kotlinx-serialization-json = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-json", version.ref = "kotlinx-serialization" }
+kotlinx-serialization-cbor = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-cbor", version.ref = "kotlinx-serialization" }
+kotlinx-serialization-hocon = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-hocon", version.ref = "kotlinx-serialization" }
+kotlinx-serialization-protobuf = { group = "org.jetbrains.kotlinx", name = "kotlinx-serialization-protobuf", version.ref = "kotlinx-serialization" }
+
+# TODO: decide whether to use kotlinx.serialization for config or konf (or maybe smth else?)
+# ktoml
+## TOML serialization for kotlinx.serialization
+ktoml-core = { group = "com.akuleshov7", name = "ktoml-core", version.ref = "ktoml" }
```

Comment:
There is a lot of stuff here apparently unused...

Reply:
yeah

I felt it might be better to put in a bunch of stuff to "look at using" and then slim it down afterwards